scaladoc/README.md

```diff
@@ -63,10 +63,17 @@ You can also find the result of building the same sites for latest `master` at:
 - <https://scala3doc.virtuslab.com/master/scala3/main/index.html>
 - <https://scala3doc.virtuslab.com/master/testcases/main/index.html>
 
+You can run the Scaladoc tool from the sbt prompt as a user would run it
+with `scaladoc/run`, passing `-d <directory>` and some `.tasty` files as
+input on the command line.
+
 ### Testing
 
-Most tests rely on comparing signatures (of classes, methods, objects etc.) extracted from the generated documentation
-to signatures found in source files. Such tests are defined using [SignatureTest](test/dotty/tools/scaladoc/signatures/SignatureTest.scala) class
+From the sbt prompt, `scaladoc/test` runs the tests.
```

Comment:
Do we also want to include that we can run specific signature tests by adding the path as the argument or is that too verbose?

```
scaladoc/test *scaladoc.signatures.MarkdownCode
```

Reply:
added